README.md

```diff
@@ -2,17 +2,19 @@
 
 # Project Name
 
-> One paragraph statement about the project.
+> This is project about palms in mauritaina
+
+## Dates Festival Tagant
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Major languages: css html js
+- Frameworks: none
+- Technologies used : vs code
 
```

Comment:
- [ ] Please record another video explaining features you implemented and Remember to highlights a piece of code you find interesting and proud of making and take your time because you have 5min maximum don't rush and keep positive because demonstrating a project shows that you understand what you did.